_docs/technical/2017-01-04-updater.md

```diff
@@ -81,13 +80,11 @@ in the future. Their meaning is briefly described below:
     of issuers of blocks with some block version is bigger than this portion,
     this block version is adopted.
 
-## Proposal Accumulation
+## Mempool Structure
 
-To vote for proposal nodes should send their
-[`votes`](https://github.com/input-output-hk/cardano-sl/blob/6b5eda44e5942599a9781e5ad3f51eb820665b83/update/Pos/Update/Core/Types.hs#L242-L253).
-Proposals are stored in a mempool (even if they don't have enough votes for inclusion into blocks,
-this way votes can be collected automatically) or gathered from the blockchain in
-order to figure out which proposal is adopted. No matter whether a change in
+`MemPool` consists of votes and proposals. Apart from that `MemState` contains tip,
+slot and `PollModifier` corresponding to `MemPool` (and to current `GState`, i. e.
+to application of `MemPool` to `GState`). No matter whether a change in
 proposal state comes from the network/mempool, or from loading
 the blockchain, `PollModifier` is stored together with mempool and it represents
```

Comment:
I think you introduced some redundancy here. The fact that `PollModifier` is stored together with `MemPool` is written twice now.